tools/accuracy_checker/accuracy_checker/metrics/semantic_segmentation.py

```diff
@@ -275,3 +276,33 @@ def reset(self):
         self.meta['names'] = names_mean + names_median
         self.meta['calculate_mean'] = False
         self.overall_metric = []
+
+class SegmentationOAR3DTiling(PerImageEvaluationMetric):
+    __provider__ = 'dice_oar3d'
+    annotation_types = (OAR3DTilingSegmentationAnnotation,)
+    prediction_types = (SegmentationPrediction,)
+
+    overall_metric = []
+
+    def configure(self):
+        self.overall_metric = []
+
+    def update(self, annotation, prediction):
+
+        eps = 1e-6
+        y_true = np.round(annotation.mask)
```

Comment:
why do you use rounding? especially for annotation?

Reply:
There are no information about dataset. All that I have is a dataset files and python notebook with dataset processing sample. DICE metric calculation code (with rounding) was implemented according to one given in sample.

Reply:
on my view, it is not looks like common practice, I think better to make it in the postprocessing.
BTW, I understand why it can be need in prediction.
Model have probabilities in the output for 2 classes - (0, 1), so for converting them to the class_id used round, please look on segmentation_one_class adapter, it contains simillar code, probaly you can use it.
For annotation, please look, which values stored in annotation, usually it is already class_ids

Reply:
Rounding has been removed